source/computer/macos-setup.rst

```diff
@@ -82,34 +70,54 @@ macOS 系统更新后，有时需重新安装 Command Line Tools for Xcode，再
 .. note::
 
    Command Line Tools for Xcode 会被安装到 :file:`/Library/Developer/CommandLineTools/`
-   目录，其提供的命令行工具位于 :file:`/Library/Developer/CommandLineTools/usr/bin` 目录，
+   目录下，其提供的命令行工具位于 :file:`/Library/Developer/CommandLineTools/usr/bin` 目录，
    包括 ``gcc``、``make``、``git`` 等。
 
 Homebrew
 ^^^^^^^^
 
 `Homebrew <https://brew.sh/index_zh-cn.html>`__ 是 macOS 下最流行的第三方软件包管理器，
-类似于 Linux 系统下的 ``apt``、``yum``、``dnf`` 等包管理器。
-其可以用于安装各种常见的软件包、库文件以及字体。日常及科研工作中所需的大多数软件、
-库文件以及字体都可以通过 Homebrew 安装。
+类似于 Linux 系统下的 ``apt``、``yum``、``dnf`` 等包管理器。安装 Homebrew 后，
+即可通过命令行的方式安装日常使用及科研工作中所需的大多数软件和库文件。
 
-安装
-""""
+安装 Homebrew
+"""""""""""""
 
-执行如下命令以安装 Homebrew::
+Homebrew 的安装脚本及相关资源托管在 `GitHub <https://github.com/>`__ 上。
+国内可能由于网络问题无法访问 GitHub，进而导致无法安装 Homebrew 或安装
+其他软件速度太慢。因而，Homebrew 的安装说明分为国内和国外两个版本。
+读者应根据自己所处的地理位置使用相应的安装说明。
 
-    $ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+.. tabbed:: 国内用户
```

Comment:
We are migrating from sphinx-panels to sphinx-design in #424. Remember to update this tab.